tests/lib/rules/sort-prop-types.js

```diff
@@ -480,6 +480,19 @@ ruleTester.run('sort-prop-types', rule, {
         });
       `,
       options: [{ callbacksLast: true, noSortAlphabetically: true }],
+    },
+    {
+      code: `
+        type Props = {
+          zzz: string;
+          aaa: string;
+        }
+        function Foo(props: Props) {
+          return null;
+        }
+      `,
+      features: ['ts', 'no-babel'],
```

Comment:
```suggestion
      features: ['types'],
```
this should work for flow too, no?

Reply:
`'no-babel'` would be required.

Reply:
why? babel is what parses flow.

Reply:
The error type for both case is different (`TSPropertySignature` and `ObjectTypeProperty`), which I have to mention in test case.

Reply:
Oops, my bad mentioning the error type isn't mandatory. Fixed it.